### PR TITLE
gz-18: Change Zendesk API access to only request and work with open tickets

### PR DESCRIPTION
**Pull request by NickMcL**
This ticket removes and modifies quite a bit of code so that the app only works with open Zendesk tickets instead of both open and closed tickets. By only focusing on the status of open Zendesk tickets, the app requires less Zendesk API calls which should increase the speed of the application as a whole.

This ticket also lays the foundation for rewriting the access to the GitHub API in order to optimize it by only requesting the needed tickets. This will probably by the task of my next ticket.

### CODE REVIEW

**Comment by NickMcL** [comment]
This ticket was supposed to be attached to issue #18, but I couldn't get it to work using hub, so that's why the ticket is labelled gz-18.

**Comment by jlward** [comment]
Next time you have a problem with hub grab on of us and we will help you work through it.

File: enhancement_tracking/templates/home.html [hunk 1]
```diff
@@ -73,26 +73,12 @@
...
-						<td class="closed">{{ a.z_date }}</td>
-					{% else %}
-						<td class="open">{{ a.z_date }}</td>
-					{% endif %}
-					{% if a.closed == 'g' %}
-						<td class="closed">{{ a.g_date }}</td>
-					{% else %}
-						<td class="open">{{ a.g_date }}</td>
-					{% endif %}
+					<td class="open">{{ a.z_date }}</td>
```
Comment (jlward) [inline enhancement_tracking/templates/home.html]: what is a? needs a better name.

File: enhancement_tracking/templates/home.html [hunk 2]
```diff
@@ -113,13 +99,51 @@
...
+				<th>Last Updated</th>
+				<th>Association Field Data</th>
+			</tr>
+		</thead>
+		<tbody>
+		{% for e in broken_enh %}
+			<tr>
+				<td>
+					<a class="open" href="{{ zen_url }}/tickets/
+						{{ e.z_id }}">Z{{ e.z_id }}</a>
```
Comment (jlward) [inline enhancement_tracking/templates/home.html]: what is e? needs a better name

File: enhancement_tracking/views.py [hunk 3]
```diff
@@ -239,19 +241,19 @@ def api_calls(request):
             else:
                 break
         
-        # Get GitHub closed tickets
+        # Get closed tickets
```
Comment (jlward) [inline enhancement_tracking/views.py]: Are we now getting all github and zendesk closed tickets? or is it still only github closed tickets? curious as to why the change in the comment.

File: enhancement_tracking/views.py [hunk 4]
```diff
@@ -259,11 +261,11 @@ def api_calls(request):
...
             else:
                 break
-        
+    
+        gticket_list = gopen_list + gclosed_list
         working['git'] = True
     except:
-        gopen_list = []
-        gclosed_list = []
+        gticket_list = []
```
Comment (jlward) [inline enhancement_tracking/views.py]: needs a better name

File: enhancement_tracking/views.py [hunk 5]
```diff
@@ -239,19 +241,19 @@ def api_calls(request):
...
                 break
         
-        # Get GitHub closed tickets
+        # Get closed tickets
         gclosed_list = []
         page = 1
-        while True:
-            r_cl = requests.get(BASE_GIT_URL % (profile.git_org,
+        while True: 
+            r_cl = requests.get(BASE_GIT_URL % (profile.git_org,
```
Comment (jlward) [inline enhancement_tracking/views.py]: needs a better name for r_cl

File: enhancement_tracking/views.py [hunk 6]
```diff
@@ -295,72 +297,35 @@ def api_calls(request):
...
+    git_tics = []
     if data_lists['status']['git']:
 
         # Filters the list of closed GitHub tickets to remove the ones that are
         # not associated with any Zendesk ticket. This filtered list is then
         # combined with all of the open GitHub tickets.
-        git_tics = []
         on_zen = []
-        for t in zen_tics_full:
+        for t in data_lists['ztics']:
```
Comment (jlward) [inline enhancement_tracking/views.py]: t needs a better name

File: enhancement_tracking/views.py (hunk 3 above)
Comment (NickMcL) [inline enhancement_tracking/views.py]: I think I changed this comment on accident, so I reverted it back to the original.

**Comment by NickMcL** [comment]
I fixed all of the naming issues that were point out in addition to improving variable naming across all of the code by merging in the name changes I just made in gz-10 as well as adding in some name changes to the code added in this ticket.
